bundles/org.openhab.io.openhabcloud/README.md

```diff
@@ -246,7 +246,7 @@ end
 rules.when().item('Apartment_Window').changed().to('OPEN').then(() => {
   actions.notificationBuilder('Apartment window was opened!')
     .withIcon('window')
-    .withSeverity('HIGH')
+    .withTag('HIGH')
```

Comment:
Can you probably change the tag value in all three examples to something like „Door“?

Just sounds better.

Reply:
done